### PR TITLE
Refactor state #6

### LINKED ISSUE #6: refactor similarities between fantasy and space state
A map should be rendered when a player enters the game

### PR DESCRIPTION
**Pull request by JonathanLiu25**
closes #6 
Refactored fantasy and space state, added/replace maps

### CODE REVIEW

**Comment by JonathanLiu25** [comment]
No conflicts and everything still works.

File: app/classes/Enemy.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+  //   this.monsterName = monsterKey
+  //   this.absorbProperties(Game.monstersInfo[monsterKey])
+  //   this.maxLife = this.life
+  //   //  Make sure this adds to Game.entities
+  //   Game.entities.add(this)
+  //   this.setAnimations(this)
+  //   this.idle(false)
+  // }
+
+  // pathfindingCallback(finalOrientation, action, delta, sendToServer, path) {
```
Comment (gabrielwr) [inline app/classes/Enemy.js]: Get rid of this since it's all on the server now.

File: app/classes/Enemy.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  //       finalOrientation = Game.computeFinalOrientation(path)
+  //       path.pop() // The player should stop right before the target, not at its location
+  //     }
+  //     var actionToSend = (action.action != 1 ? action : { action: 0 })
+  //     if (this.isPlayer && sendToServer && path.length) Client.sendPath(path, actionToSend, finalOrientation)
+  //     this.move(path, finalOrientation, action, delta)
+  //   }
+  // }
+
+  // setupMovement(path, action, delta) {
```
Comment (gabrielwr) [inline app/classes/Enemy.js]: and this.

File: app/classes/Enemy.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  //       this.tween.stop()
+  //       this.tween = null
+  //     }
+  //     this.orientation = direction
+  //     //  this.attack is contained in entityPrefab
+  //     this.attack()
+  //   }
+  // }
+
+  // adjacent(a, b) {
```
Comment (gabrielwr) [inline app/classes/Enemy.js]: we probably don't need this anymore either?

**Review by gabrielwr — CHANGES_REQUESTED** [review]
see comments. Looks good otherwise.

**Comment by JonathanLiu25** [comment]
You're right, I removed code that is currently unused and doesn't affect what we have now.

**Pull request merged**